### PR TITLE
Added example for onclick

### PR DESCRIPTION
**Pull request by vikrantuk**
Added @types/applepayjs as dev dependency for type proofing.
Added complete working example for ApplePaySession

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,8 +29,111 @@ import { ApplePayButton } from 'apple-pay-button';
...
+      const paymentData = event.payment;
+
+      if (paymentData.token) {
+        // Forward token to your gateway for processing payment and return result to apple pay session
+        const result: ApplePayJS.ApplePayPaymentAuthorizationResult = {
+          status: ApplePaySession.STATUS_SUCCESS,
+        };
+        session.completePayment(result);
+      }
+      else {
```
Comment (johan44co) [inline README.md]:
```suggestion
      } else {
```

**Review by johan44co — APPROVED** [review]
Awesome, @vikrantuk thanks for the contribution!

**Review by johan44co — APPROVED** [review]
@vikrantuk Let's merge!

**Comment by vikrantuk** [comment]
> @vikrantuk Let's merge!

@johan44co Sure, thanks

**Pull request merged**